### DISCUSSION TITLE
Why is wrapping disabled? How do people work without wrapping?

### DISCUSSION THREAD

**Discussion by briandipalma** [LazyVim/LazyVim#349] [General]
Before LazyVim I had wrapping setup but I thought I'd give the vanilla LazyVim experience a go before seeing if I needed any of my previous configuration so I threw all the wrapping configuration away.

Anyway after code reviewing and finding longer text strings very awkward to read and work with I've given up and added back my previous wrapping configuration. Below is what it looks like for i18n strings which are one of the few types of documents that require wrapping for me. You can see I have return characters at the start of the wrapped line which is slightly indented and only whole words are wrapped. `j`/`k` act as I expect. I don't feel any pain/friction with wrapping enabled. It's easier to review a small change in a long line like this for me (in this case the entire line is added but still it's easier to review/work with it as I can see it without having to jump horizontally).

![image](https://user-images.githubusercontent.com/1597820/221872419-1a93c35d-c766-4f7a-aa92-24efe89a0457.png)

It makes me feel I'm missing something when it comes to wrapping, is there some feature people use that makes wrapping redundant? Is it not awkward to have parts of longer strings/sentences hidden? Do people have keymaps that toggle it on and off? What's bad about wrapping?

**Comment by folke** [comment]
I personally don't like (nor need) wrapping, so it's just a personal preference. Some people like it, others don't

**Comment by ChenRuiwei** [comment]
LazyVim has a default shortcut key for toggling the Wrap option. See if this is helpful to you.

<img width="453" alt="Snipaste_2023-03-04_22-25-36" src="https://user-images.githubusercontent.com/92102709/222908031-8b0e9dc0-5f7e-484f-a134-0225915a8f6d.png">
